classes/tombstone/DataObjectTombstoneSettingsDAO.inc.php

```diff
@@ -34,7 +34,7 @@ function &getSetting($tombstoneId, $name, $locale = null) {
 		while (!$result->EOF) {
 			$row = $result->getRowAssoc(false);
 			$value = $this->convertFromDB($row['setting_value'], $row['setting_type']);
-			if ($row['locale'] == '') $setting[$name] = $value;
+			if (!array_key_exists('locale', $row) || $row['locale'] == '') $setting[$name] = $value;
```

Comment:
Maybe I'm wrong, but I thought `empty($array['key'])` was safe to call even if the array key doesn't exist...